dev/com.ibm.websphere.rest.handler/src/com/ibm/wsspi/rest/config/ConfigBasedRESTHandler.java

```diff
@@ -134,13 +134,21 @@ private static String getUID(String configDisplayId, String id) {
     public abstract Object handleError(RESTRequest request, String uid, String errorMessage);
 
     /**
-     * Common implementation of handleRequest which locates configured instances matching the request,
-     * delegating each to handleSingleInstance, and, after all of which have completed, invokes createResponse.
+     * Common implementation of handleRequest which limits ability to make only GET requests, locates configured instances
+     * matching the request, delegating each to handleSingleInstance, and, after all of which have completed, invokes
+     * createResponse.
      *
      * @see com.ibm.wsspi.rest.handler.RESTHandler#handleRequest(com.ibm.wsspi.rest.handler.RESTRequest, com.ibm.wsspi.rest.handler.RESTResponse)
      */
     @Override
     public final void handleRequest(RESTRequest request, RESTResponse response) throws IOException {
+
+        if (!"GET".equals(request.getMethod())) {
```

Comment:
We shouldn't put the limitation here because it will limit all REST API that is based on server config to only allowing GET.  Instead, it should be place on the specific REST endpoints (config and validation) that we are trying to limit. And yes, I realize that config & validation are current the only config-based REST endpoints, but the ConfigBasedRESTHandler is intended for more general use in the future, and this would limit it.